docs/the-new-architecture/pillars.md

```diff
@@ -9,24 +9,31 @@ import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';
 
 The New Architecture is composed mainly by two pillars:
 
-- [TurboModules](pillars-turbomodules)
-- [Fabric Components](pillars-fabric-components).
+- [The New Native Module System - TurboModules](pillars-turbomodules)
+- [The New Renderer - Fabric](pillars-fabric-components).
 
-TurboModules are the preferred way to create libraries that leverage some platform specific API. Fabric Components are the preferred way to create reusable UI components, providing a native experience to the users.
+The core concepts of React Native still holds true in the New Architecture: Native Modules are the preferred way to create libraries that leverage some platform specific API. Native Components are the preferred way to create reusable UI components, providing a native experience to the users.
 
-The main goal of this section is to drive the reader through a step-by-step guide to create their first TurboModule or Fabric Component.
+The main goal of this section is to drive the reader through a step-by-step guide to create their first Native Module or Component which is compatible with the New Architecture.
+
+:::info
+For the sake of this guide we're going to use the following **terminology**:
+
+- **Legacy Native Components** & **Legacy Native Modules** - To refer to Modules and Components which are running on the old React Native architecture.
+- **Fabric Native Components** & **Turbo Native Modules** - To refer to Modules and Components which have been adapted to work well with the New Architecture, namely the new renderer and the new Native Module System. For brevity you might find them referred as **Fabric Components** and **Turbo Modules**
+  :::
 
 The next sections contain an high-level overview of the pillars, together with the steps to create them. To create one of these pillars, the steps are:
 
 1. Define a JavaScript specification using Flow or TypeScript.
-1. Configure the dependencies management system to generate code from the provided spec.
-1. Implement the Native code.
-1. Integrate the code in the app.
+2. Configure the dependencies management system to generate code from the provided spec.
+3. Implement the Native code.
+4. Integrate the code in the app.
```

Comment:
Isn't docusaurus able to resolve the numbers when it renders the page?

Reply:
I think this was my VS Code trying to be smart